src/cmd/manage_nsfs.js

```diff
@@ -353,25 +340,64 @@ async function account_management(argv, config_root, from_file) {
     await manage_account_operations(action, data, config_root, config_root_backend);
 }
 
+/**
+ * set_access_keys will set the access keys either given as args or generated.
+ * @param {{ access_key: any; secret_key: any; }} argv
+ * @param {boolean} generate a flag for generating the access_keys automatically
+ */
+function set_access_keys(argv, generate) {
+    const { access_key, secret_key } = argv;
+    let generated_access_key;
+    let generated_secret_key;
+    if (generate) {
+        ({ access_key: generated_access_key, secret_key: generated_secret_key } = cloud_utils.generate_access_keys());
+        generated_access_key = generated_access_key.unwrap();
+        generated_secret_key = generated_secret_key.unwrap();
+    }
+
+    return [{
+        access_key: access_key || generated_access_key,
+        secret_key: secret_key || generated_secret_key,
+    }];
+}
+
+/**
+ * Will print the access_keys
+ * @param {{ access_keys: any[]; }} data
+ */
+function print_access_keys(data) {
```

Comment:
on my [PR](https://github.com/noobaa/noobaa-core/pull/7655) i'm handling writing to stdout and writing access_keys on update/add always and on list and status only if --show_secrets flag was provided so I think you can remove it